build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
```

Comment:
Might want to add `set -e` so any failure gets returned to the called of the script